### PR TITLE
Skip integration tests when running rspec command.

### PR DESCRIPTION
**Pull request by tingulfsen**
This is intended to get Travis to succeed, as well as skip integration tests when running the regular rspec command.

### CODE REVIEW

File: spec/integration/bulk_jobs_spec.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@
                   'xlsx',
                   'true',
                   'anote')
-
+puts "tommy: here2"
```
Comment (mejackreed) [inline spec/integration/bulk_jobs_spec.rb]: Is this `puts` statement leftover cruft?

**Comment by mejackreed** [comment]
:+1: besides my comment about the `puts` statement

**Comment by tingulfsen** [comment]
Thanks for catching that, I shouldn't be leaving that stuff around.

**Comment by jmartin-sul** [comment]
:+1:

**Pull request merged**